bedrock/products/tests/test_helper_misc.py

```diff
@@ -0,0 +1,73 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from django.test.client import RequestFactory
+from django.test.utils import override_settings
+
+from django_jinja.backend import Jinja2
+
+from bedrock.mozorg.tests import TestCase
+
+
+TEST_FXA_ENDPOINT = 'https://accounts.firefox.com/'
+
+jinja_env = Jinja2.get_default()
+
+
+def render(s, context=None):
+    t = jinja_env.from_string(s)
+    return t.render(context or {})
+
+
+@override_settings(FXA_ENDPOINT=TEST_FXA_ENDPOINT)
+class TestVPNSubscribeLink(TestCase):
+    rf = RequestFactory()
+
+    def _render(self, entrypoint, link_text, class_name=None, optional_parameters=None, optional_attributes=None):
+        req = self.rf.get('/')
+        req.locale = 'en-US'
+        return render("{{{{ vpn_subscribe_link('{0}', '{1}', '{2}', {3}, {4}) }}}}".format(
+                      entrypoint, link_text, class_name, optional_parameters, optional_attributes),
+                      {'request': req})
+
+    def test_vpn_subscribe_link(self):
+        """Should return expected markup"""
+        markup = self._render(entrypoint='mozilla.org', link_text='Try Mozilla VPN',
+                              class_name='vpn-button try js-try-vpn',
+                              optional_parameters={'utm_campaign': 'vpn-landing-page'},
+                              optional_attributes={'data-cta-text': 'Try Mozilla VPN', 'data-cta-type':
+                                                   'fxa-vpn', 'data-cta-position': 'primary'})
+        expected = (
+            u'<a href="https://vpn.mozilla.org/r/vpn/subscribe/?entrypoint=mozilla.org&form_type=button'
```

Comment:
In other spots where we set entrypoint params for FxA, we use `entrypoint=mozilla.org-{service/UID}`. Should we add a `-vpn`, etc. here? 

From vpn.moz.org, they use `entrypoint=www.mozilla.org-vpn-product-page`. It may make sense to use the same ID here for joining Amplitude analytics. 

```suggestion
            u'<a href="https://vpn.mozilla.org/r/vpn/subscribe/?entrypoint=mozilla.org-vpn&form_type=button'
```

Other references: 
<img width="1006" alt="image" src="https://user-images.githubusercontent.com/2692333/108174335-eef70880-70c4-11eb-8fa9-af3dd17023a9.png">

Reply:
On the landing page we're using `www.mozilla.org-vpn-product-page` for entrypoint. On other pages we'll use things like `www.mozilla.org-vpn-faq-page`, `www.mozilla.org-vpn-comparison-page`. I'll update this test example to follow suit 👍